components/community/Card.tsx

```diff
@@ -0,0 +1,82 @@
+import {
+  Box,
+  Flex,
+  LinkBox,
+  LinkOverlay,
+  Stack,
+  VStack,
+} from "@chakra-ui/react";
+import { ChakraNextImage } from "components/Image";
+import { Heading, Text } from "tw-components";
+
+interface CommunityCardProps {
+  image: string;
+  link: string;
+  title: string;
+  description: string;
+}
+
+export const CommunityCard: React.FC<CommunityCardProps> = ({
+  image,
+  link,
+  title,
+  description,
+}) => {
+  return (
+    <LinkBox
+      bg="transparent"
+      borderRadius="xl"
+      w="full"
+      border="1px solid"
+      borderColor="borderColor"
+    >
+      <VStack
+        bg="transparent"
+        borderRadius="xl"
+        as={LinkOverlay}
+        href={link}
+        _hover={{ textDecor: "none" }}
+      >
+        {image ? (
+          <Stack
+            align="center"
+            justify="center"
+            w="full"
+            bg="#181818"
+            h="11.5rem"
+          >
+            <ChakraNextImage
+              alt={title}
+              borderTopRadius="xl"
+              boxSize="100%"
+              objectFit="cover"
+              src={image}
+              width={90}
+              height={90}
+            />
+          </Stack>
+        ) : (
+          <Box
+            borderTopRadius="xl"
+            boxSize="100%"
+            w="full"
+            bg="linear-gradient(180deg, #893AA1 0%, #BFA3DA 100%)"
+            h="11.5rem"
+          />
+        )}
```

Comment:
they should all require an image without an image this card (based on the design of the page) is not valid so this is extra code for nothing